staging/src/k8s.io/kube-apiextensions-server/test/integration/basic_test.go

```diff
@@ -76,14 +88,14 @@ func TestSimpleCRUD(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	noxuNamespacedWatch, err := noxuNamespacedResourceClient.Watch(metav1.ListOptions{ResourceVersion: initialListListMeta.GetResourceVersion()})
+	noxuWatch, err := noxuResourceClient.Watch(metav1.ListOptions{ResourceVersion: initialListListMeta.GetResourceVersion()})
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer noxuNamespacedWatch.Stop()
+	defer noxuWatch.Stop()
 
 	noxuInstanceToCreate := testserver.NewNoxuInstance(ns, "foo")
-	createdNoxuInstance, err := noxuNamespacedResourceClient.Create(noxuInstanceToCreate)
+	createdNoxuInstance, err := noxuResourceClient.Create(noxuInstanceToCreate)
```

Comment:
Also check that resources with the wrong scope are rejected.

And check that creating another resource with another namespace, but same name works or fails (depending on scope).